README.md

```diff
@@ -116,6 +117,55 @@ response = client.market_data.trades.get_latest(
 )
 ```
 
+### Streaming API Usage
+```python
+from lemon import api
+
+client = api.create(...)
+
+# get live streaming authentication token
+response = client.streaming.authenticate()
+```
+
+#### Streaming API example
+This example relies on that you have both this SDK installed as well as paho-mqtt package.
+
+Below is an example usage of live streaming quotes through alby mqtt broker using paho mqtt client.
+When connecting to the broker the on_connect callback will be triggered.
+This in return will trigger the on_subscribe callback where we can let the broker know what ISINS we are interested in
+
+After that we will simply get all the quote updates through the on_message callback.
+
+```python
+from lemon import api
+import paho.mqtt.client as mqtt
+
+client = api.create(...)
+
+# get live streaming authentication token
+response = client.streaming.authenticate()
+
+def on_connect(mqtt_client, userdata, flags, rc):
+    mqtt_client.subscribe(response.user_id)
+
+def on_subscribe(mqtt_client, userdata, level, buff):
+    mqtt_client.publish(f"{response.user_id}.subscriptions", "ISIN1,ISIN2,ISIN...")
```

Comment:
1. I would use here some real isin values
2. Is the dots at the end are not required - remove them pls.